src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -5,13 +5,17 @@ namespace System.Diagnostics
 {
     public partial class Process
     {
+        // The proc_start_time RUsage is in nanoseconds but we need to convert to milliseconds so
+        // this is the number of nanoseconds in a millisecond.
+        private const ulong NumberOfNanoSecondsInMilliseconds = 100000000000;
```

Comment:
I'm not sure if the comment and field name are incorrect or if the value is incorrect, but there are 1,000,000 nanoseconds in a millisecond, not 100,000,000,000.